source/slang/slang-extension-usage-tracker.h

```diff
@@ -0,0 +1,34 @@
+// slang-extension-usage-tracker.h
```

Comment:
Everything relevant to tracking extension usage probably belongs in `emit-glsl.*` or `emit-glsl-*` since it is all going to be GLSL-specific (I guess eventually we might need a SPIR-V equivalent, but we can deal with that when it comes up.